test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/MiddlewareFilterConfigurationProviderTest.cs

```diff
@@ -14,6 +14,19 @@ namespace Microsoft.AspNetCore.Mvc.Internal
     public class MiddlewareFilterConfigurationProviderTest
     {
         [Fact]
+        public void CreateConfigureDelegate_ThrowsIfNoConstructorFoundForType()
```

Comment:
`CreateConfigureDelegate_ThrowsIfTypeIsAbstract`

Need another one where the type is non-abstract but does not have a parameterless ctor.